documentation/airport-guides/lszh.md

```diff
@@ -0,0 +1,51 @@
+### About this facility
+Skyguide Air Navigation Center, Dübendorf, Switzerland
+269,160 annual operations in 2016
+
+Located at Dübendorf Air Base (close to Zurich), besides the LSAZ FIR, the Skyguide Air Navigation Center is also responsible for the in- and outbound traffic of Switzerlands major airport Zurich (LSZH).
+
+### Airspace
+The airspace area extends up to 55nm of distance to the airport and is also covering parts of German national territory. Due to the fact that approaches to runway 14 and 16 require the use of German airspace, these approaches aren't available during the German ordinance period. (MON - FRI: 2100-0700LT; SAT, SUN, holidays;  2000-0700LT). Therefore runway 28 or 34 is usually used for landings then which is reducing the capacity of the airport. Vertically the airspace is covering aircraft up to FL190.
+
+### Instrument Procedures
+For outbound traffic LSZH offers a couple of SIDs eventhough usually only DEGES and VEBIT are used. Inbound traffic follows a STAR to either GIPOL or AMIKI. Inbound traffic must be vectored to their final approach course.
+
+### Sectors Available in OpenScope
+```
+D - Zurich Departure
+W - Zurich Arrival (West)
+E - Zurich Arrival (East)
+A - Zürich Arrival (ARFA)
```

Comment:
Do you know what `ARFA` stands for? I couldn't find anything

Reply:
I've looked it up now and ARFA is responsible for the approaches to LSZR (Altenrhein = AR) and EDNY (Friedrichshafen = FA) => AR + FA = ARFA
Sector can be seen here: [https://image.prntscr.com/image/XoIX0r4IQOawbjqqH0O2aw.png](https://image.prntscr.com/image/XoIX0r4IQOawbjqqH0O2aw.png). ARFA is up to FL100 and above (for inbound traffic to LSZH) its covered by Zurich Arrival east sector.
I should delete it then, shouldn't I?

Reply:
If it's covered by the East sector yes, but if the ARFA sector is partly responsibly for LSZH, then keep it.